<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkbackView</class>
 <widget class="QDialog" name="TalkbackView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>      对讲操作</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="talkback_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>对讲测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>静音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allow_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>允许发言</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>结束发言</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>移除请求标志</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="popup_chex">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学员请求时对讲自动弹出</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>620</width>
     <height>250</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>618</width>
      <height>248</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chose_all">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>681</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
